test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/035_ATF_P_TC_media_app_PTU_fail_ForceProtectedService_OFF_RPC_encrypted_true.lua

```diff
@@ -0,0 +1,167 @@
+--
+-- Description:
+-- In case SDL starts PolicyTableUpdate in case of no "certificate" at "module_config" section at LocalPT
+-- and PolicyTableUpdate is failed by any reason even after retry strategy
+-- and "ForceProtectedService" is OFF at .ini file
+-- and app sends StartService (<any_serviceType>, encypted=true) to SDL
+-- SDL must respond StartService (ACK, encrypted=false) to this mobile app
+--
+-- 1. Used preconditions:
+-- RPC SetAudioStreamingIndicator is allowed by policy
+-- ForceProtectedService is set to OFF in .ini file
+-- Media app exists in LP, no certificate in module_config
+-- Register and activate application.
+-- Send StartService(serviceType = 7 (RPC), RPCfunctionID = 48(SetAudioStreamingIndicator))
+-- -> SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- -> SDL should not respond to StartService_request
+-- -> SDL should not process request to HMI
+--
+-- 2. Performed steps
+-- Send wrong policyfile
+--
+-- Expected result:
+-- SDL must respond StartService (ACK, encrypted=false) to this mobile app
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"MEDIA"}
+config.application1.registerAppInterfaceParams.isMediaApplication = true
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForPolicyCeritificates = require('user_modules/shared_testcases/testCasesForPolicyCeritificates')
+local events = require('events')
+local Event = events.Event
+
+--[[ Local variables ]]
+local rpc_corrid
+
+--[[ General Precondition before ATF start ]]
+commonPreconditions:BackupFile("smartDeviceLink.ini")
+commonFunctions:write_parameter_to_smart_device_link_ini("ForceProtectedService", "Non")
+testCasesForPolicyCeritificates.update_preloaded_pt(config.application1.registerAppInterfaceParams.appID, false)
+testCasesForPolicyCeritificates.create_ptu_certificate_exist(false, true)
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+function Test:TestStep_First_StartService()
```

Comment:
@dboltovskyi, updated as Precondition in 15bbd6556305449a954dde0b03d619414cd81597